api/management.openapi.yaml

```diff
@@ -0,0 +1,243 @@
+openapi: 3.0.0
+info:
+  title: Cartesi Rollups Node Management API
+  description: |-
+    This is a management API for the Cartesi Rollups Node. It allows to dynamically add and remove applications managed by the node, or query the list of managed applications.
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
+tags:
+  - name: applications
+    description: Cartesi applications
+paths:
+  /applications:
+    get:
+      tags:
+        - applications
+      summary: Fetches list of applications managed by the node
+      description: List of managed applications
+      operationId: getApplications
+      parameters:
+        - name: status
+          in: query
+          description: Filter by application status
+          schema:
+            $ref: "#/components/schemas/ApplicationStatus"
+        - name: offset
+          in: query
+          description: Items to skip for pagination
+          required: false
+          schema:
+            type: integer
+            minimum: 0
+            default: 0
+        - name: limit
+          in: query
+          description: Maximum number of items per page
+          required: false
+          schema:
+            type: integer
+            minimum: 1
+            default: 100
+      responses:
+        "200":
+          description: Success
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/PaginatedResult"
+                  - type: object
+                    properties:
+                      data:
+                        type: array
+                        items:
+                          $ref: "#/components/schemas/Application"
+                    required:
+                      - data
+  /applications/{address}:
+    get:
+      tags:
+        - applications
+      summary: Find application by address
+      description: Returns a single application by its address
+      operationId: getApplicationByAddress
+      parameters:
+        - name: address
+          in: path
+          description: Address of application to return
+          required: true
+          schema:
+            $ref: "#/components/schemas/Address"
+      responses:
+        "200":
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Application"
+        "400":
```

Comment:
Could you reuse common responses? See https://swagger.io/docs/specification/describing-responses/#reuse

Reply:
Done